llvm/lib/Target/LoongArch/LoongArchCallLowering.cpp

```diff
@@ -0,0 +1,52 @@
+//===-- LoongArchCallLowering.cpp - Call lowering ---------------*- C++ -*-===//
```

Comment:
Since the filename is too generic, you can put this file and other files in a seperate sub-folder like `GISel`.